xml/System.Threading/Interlocked.xml

```diff
@@ -1341,22 +1341,21 @@ If `comparand` and the object in `location1` are equal by reference, then `value
         <Parameter Name="comparand" Type="T" Index="2" FrameworkAlternate="dotnet-uwp-10.0;net-10.0;net-5.0;net-6.0;net-7.0;net-8.0;net-9.0;netcore-1.0;netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netcore-3.1;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;netframework-4.8.1;netstandard-1.0;netstandard-1.1;netstandard-1.2;netstandard-1.3;netstandard-1.4;netstandard-1.6;netstandard-2.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <typeparam name="T">The type to be used for <paramref name="location1" />, <paramref name="value" />, and <paramref name="comparand" />. This type must be a reference type.</typeparam>
```

Comment:
This doc update applies to .NET 9+ only. Do we want to capture that?

Reply:
Maybe by adding NotSupportedException to the list of exceptions this might throw. It can say that on .NET 9 and later, the T is allowed to be types other than reference types, but it will throw if the T isn't supported, in particular for types that aren't reference, primitive, or enum types.